ubccr/coldfront
# Notifications for warranty and service expirations on resources

Resolved #371 : Provided email and webpage message notifications for resource warranty and service attribute expirations based on a user defined set of days similar to how 'EMAIL_ALLOCATION_EXPIRING_NOTIFICATION_DAYS' works. Also, users have the option to disable email notifications with 'EMAIL_RESOURCE_NOTIFICATIONS_ENABLED' (defaulted to off). 

Tested:
1. Set end dates for warranty and service attributes for resources with and without children resources.
2. Ensure that dates that are less than or equal to the set dates in 'EMAIL_RESOURCE_EXPIRING_NOTIFICATION_DAYS' will cause a message to appear on the webpage when viewing the resource on the resource detail page.
3. Ensure that emails are sent (if enabled) when end dates are equal to those set in 'EMAIL_RESOURCE_EXPIRING_NOTIFICATION_DAYS'.
4. Ensure webpages work for resources that do not have service and warranty end dates set or their end dates are not soon enough to generate notifications.
